openslides/motions/models.py

```diff
@@ -697,6 +698,57 @@ def get_root_rest_element(self):
         return self.motion
 
 
+class MotionChangeRecommendation(RESTModelMixin, models.Model):
+    """
+    A MotionChangeRecommendation object saves change recommendations for a specific MotionVersion
+    """
+
+    access_permissions = MotionChangeRecommendationAccessPermissions()
+
```

Comment:
Please add a custom manager with the get_full_queryset method, see most other models. This method is to improve performance during cache filling.